docs/docs/Components/components-data.mdx

```diff
@@ -187,6 +187,64 @@ The output of the **File** component depends on the number and type of files loa
 
 - **Multiple files**: Produces a **Files** [`DataFrame`](/data-types#dataframe) containing the content and metadata of all selected files.
 
+## Create Knowledge
+
+The **Create Knowledge** component creates or appends to a [Langflow Knowledge Base](/knowledge) from a DataFrame.
+
+To create a knowledge base in a flow, see the **Create Knowledge** template.
+
+For more information, see the [Knowledge Base documentation](/knowledge).
```

Comment:
TBH, I would do what I did for the [Agent and MCP components](https://d5rxiv0do0q3v.cloudfront.net/langflow-drafts/docs-knowledge-base-feature/components-agents): Put everything on the /knowledge page, and then just point to that page from here. Then the user doesn't have to pop between two pages to read how to use it in a flow and learn about the parameters.

```suggestion
For more information, see [Manage knowledge bases](/knowledge).
```

Move line 198 to 219 to the /knowledge page.